### DISCUSSION TITLE
Make a Button on results page to copy the result

### DISCUSSION THREAD

**Discussion by Kavikara** [Yue-Yang/ChatGPT-Siri#21] [Ideas]
Can you update this code with a Button on the results page to copy the text inside the result? It is really helpful for me and others.

**Comment by Yue-Yang** [comment]
> Can you update this code with a Button on the results page to copy the text inside the result? It is really helpful for me and others.

If you start using Siri, you cannot add a button. You can choose to use the click method, which prompts a dialogue box with the results that can be copied. Alternatively, after ending the conversation, you can check all chat records in the notes or files.
